netctl/netctl.go

```diff
@@ -619,7 +619,9 @@ func createEndpointGroup(ctx *cli.Context) {
 }
 
 func inspectEndpointGroup(ctx *cli.Context) {
-	argCheck(1, ctx)
+	if len(ctx.Args()) != 1 {
+		errExit(ctx, exitHelp, "endpoint group name is required", true)
+	}
 
```

Comment:
Please remove this change. I have another PR https://github.com/contiv/netplugin/pull/539 
 for this and some other stuff.